### ISSUE TITLE
Make Return Output function only pick the direction out

### ISSUE THREAD

**Issue by mildronize** [mildronize/nammatham#46]
Previously, we need to use `Partial` type to wrap the `GetBindings` type, for multiple return type

```ts
// 0.7.0
import { BaseFunction, Binding, functionName, GetBindings } from 'nammatham';

const bindings = [
  Binding.httpTrigger({ name: 'req' as const}),
  Binding.http({ name: 'res' as const }),
];


@functionName('WithTypeUtility', ...bindings)
export class WithTypeUtilityFunction extends BaseFunction<typeof bindings> {

  public override execute(): Partial<GetBindings<typeof bindings>> {
    const { name } = this.req.query;
    return {
      res: this.res.send()
    }
  }
}
```

multiple outputs Return binding:

https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=v2-v3-v4-export%2Cv2-v3-v4-done%2Cv2%2Cv2-log-custom-telemetry%2Cv2-accessing-request-and-response%2Cwindows-setting-the-node-version#outputs

as shown in the doc:

```js
module.exports = async function(context) {
    let retMsg = 'Hello, world!';
    return {
        httpResponse: {
            body: retMsg
        },
        queueOutput: retMsg
    };
};
```

## Expected Behavior 

We want the `GetOutputBindings` to remove all `input` direction

```ts
type GetOutputBindings<T  extends readonly FunctionBinding<unknown>[]> = Pick<GetBindings<T>, 'res'>;
```

So, we can you like this: 

```ts
import { BaseFunction, Binding, functionName, GetOutputBindings } from 'nammatham';

const bindings = [
  Binding.httpTrigger({ name: 'req' as const}),
  Binding.http({ name: 'res' as const }),
];


@functionName('WithTypeUtility', ...bindings)
export class WithTypeUtilityFunction extends BaseFunction<typeof bindings> {

  public override execute(): GetOutputBindings<typeof bindings> {
    const { name } = this.req.query;
    return {
      res: this.res.send()
    }
  }
}
```

## Todo
- [ ] Add Doc for multiple return bindings

**Comment by mildronize** [comment]
I've done PoC on `main.issues-46`

**Comment by mildronize** [comment]
type utils

```ts
binding.inferReturn<typeof bindings>
binding.inferOutput<typeof bindings>
binding.inferInput<typeof bindings>
binding.infer<typeof bindings>
```

**Comment by mildronize** [comment]
Already fixed in #61
